lxd/db/cluster/auth_groups.go

```diff
@@ -106,14 +106,9 @@ func (g *AuthGroup) ToAPI(ctx context.Context, tx *sql.Tx) (*api.AuthGroup, erro
 		return nil, err
 	}
 
-	group.Identities = make([]api.Identity, 0, len(identities))
+	group.Identities = make(map[string][]string, 2)
```

Comment:
why 2 here out of interest?

Reply:
I guess I was thinking there are currently only to authentication methods that are used in the identities table. It doesn't make sense to add it here though as we may add more. It makes more sense to preallocate the slices to `len(identities)`. Will change.